app/components/work_packages/activities_tab/journals/item_component/reactions.html.erb

```diff
@@ -3,29 +3,32 @@
     if grouped_emoji_reactions.present?
       flex_layout(test_selector: "emoji-reactions", flex_wrap: :wrap, classes: "op-emoji-reactions--gap") do |reactions_container|
         grouped_emoji_reactions.each do |reaction, data|
-          reactions_container.with_column(mr: 2) do
-            render(Primer::Beta::Button.new(
-                    scheme: button_scheme(data[:users]),
-                    color: :default,
-                    bg: counter_color(data[:users]),
-                    id: "#{journal.id}-#{reaction}",
-                    test_selector: "reaction-#{reaction}",
-                    tag: :a,
-                    href: href(reaction:),
-                    data: {
-                      turbo_stream: true,
-                      turbo_method: :put,
-                      "work-packages--activities-tab--index-target": "reactionButton"
-                    },
-                    aria: { label: aria_label_text(reaction, data[:users]) },
-                    disabled: current_user_cannot_react?,
-                    classes: "op-reactions-button"
-                  )) do |button|
-              button.with_tooltip(text: number_of_user_reactions_text(data[:users]),
-                                  test_selector: "reaction-tooltip-#{reaction}") do
-                button.with_icon(EmojiReactions.emoji(reaction), size: :small)
-              end
-              "#{EmojiReaction.emoji(reaction)} #{data[:count]}"
+          reactions_container.with_column(mr: 1) do
+            render(
+              Primer::Beta::Button.new(
+                scheme: button_scheme(data[:users]),
+                color: :default,
+                bg: counter_color(data[:users]),
+                id: "#{journal.id}-#{reaction}",
+                test_selector: "reaction-#{reaction}",
+                tag: :a,
+                href: href(reaction:),
+                data: {
+                  turbo_stream: true,
+                  turbo_method: :put,
+                  "work-packages--activities-tab--index-target": "reactionButton"
+                },
+                aria: { label: aria_label_text(reaction, data[:users]) },
+                disabled: current_user_cannot_react?,
+                classes: %w[op-reactions-button f4 px-2]
```

Comment:
Please do not use these classes directly. Instead use the `system_arguments` of Primer. Thus, we can be sure that the code works even if they change the class names in the future